Add troubleshooting note to disable DataStore when using API category (#2115)
aws-amplify/docs: docs/lib/graphqlapi/fragments/native_common/getting-started/common.md

    `No`
```

<amplify-callout warning>

> **Troubleshooting:** The API plugin does not support conflict detection, so if AppSync returns errors about missing `_version` and `_lastChangedAt` fields, or unhandled conflicts, disable **conflict detection** by running `amplify update api`, and then choose **Disable DataStore for entire API**.  This step is likely required if you started with the Getting Started guide, which starts with DataStore.

Reviewer: ```suggestion
**Troubleshooting:** The AWS API plugins do not support conflict detection. If AppSync returns errors about missing `_version` and `_lastChangedAt` fields, or unhandled conflicts, disable **conflict detection**. Run `amplify update api`, and choose **Disable DataStore for entire API**.  If you started with the Getting Started guide, which introduces DataStore, this step is required.
```
